build/scripts/crossgen.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+set -e
+
+BIN_DIR="$( cd $1 && pwd )"
```

Comment:
I'm fine if we don't harden it but can we add a comment as to what `$1` should be?